notebooks/regression/tutorial_regression.md

```diff
@@ -205,11 +229,14 @@ _ = ax.legend(fontsize=10, loc=[1, 0.4])
 As expected, the prediction intervals estimated by the Naive method
 are slightly too narrow. The Jackknife, Jackknife+, CV, CV+, JaB, and J+aB give
 similar widths that are very close to the true width. On the other hand,
-the widths estimated by Jackknife-minmax and CV-minmax are slightly too
+the width estimated by Jackknife-minmax and CV-minmax are slightly too
 wide. Note that the widths given by the Naive, Jackknife, and CV strategies
 are constant because there is a single model used for prediction,
 perturbed models are ignored at prediction time.
 
+It's interesting to observe that quantile strategy offers more varying width,
+often giving much higher but also lower interval width than other methods.
```

Comment:
Jackknife and CV strategies should be prefered when dealing with homoscedastic noise ?

Reply:
True, I will explain that CQR should not be the preferred method in this case.